examples/varianter_cit/params.ini

```diff
@@ -1,6 +1,18 @@
-[parameters]
-color: black, gold, red, green
-shape: square, triangle, circle
-state: liquid, solid, gas
-material: leather, plastic, aluminum
-coating: anodic, cathodic
+
```

Comment:
Is there a reason for the empty line at the beginning? I found it confusing, and after removing it I didn't notice any change.  I think we should remove it as this is a sample for other people writing their own configuration files.

Also, this file format doesn't comply with that is a INI file:

https://en.wikipedia.org/wiki/INI_file

It doesn't have sections, and `key=value` attributes.  I suggest we change the file suffix to something else (could be as generic as `.conf` or as specific as `.cit`.

Reply:
Yes, I missed that empty line and I will remove it. Also, it is better to use our file suffix. I will change that to `.cit`.